t/mojo/user_agent_tls.t

```diff
@@ -27,46 +27,56 @@ my $listen
   . '&ca=t/mojo/certs/ca.crt&verify=0x03';
 my $port = $daemon->listen([$listen])->start->ports->[0];
 
-# No certificate
-my $ua = Mojo::UserAgent->new(ioloop => Mojo::IOLoop->singleton);
-my $tx = $ua->get("https://127.0.0.1:$port");
-ok $tx->error, 'has error';
-$tx = $ua->get("https://127.0.0.1:$port");
-ok $tx->error, 'has error';
-$tx = $ua->ca('t/mojo/certs/ca.crt')->get("https://127.0.0.1:$port");
-ok $tx->error, 'has error';
-$tx = $ua->get("https://127.0.0.1:$port");
-ok $tx->error, 'has error';
-
-# Valid certificates
-$ua->ca('t/mojo/certs/ca.crt')->cert('t/mojo/certs/client.crt')->key('t/mojo/certs/client.key');
-$tx = $ua->get("https://127.0.0.1:$port");
-ok !$tx->error, 'no error';
-is $tx->res->code, 200,      'right status';
-is $tx->res->body, 'works!', 'right content';
-
-# Valid certificates (env)
-$ua = Mojo::UserAgent->new(ioloop => $ua->ioloop);
-{
-  local $ENV{MOJO_CA_FILE}   = 't/mojo/certs/ca.crt';
-  local $ENV{MOJO_CERT_FILE} = 't/mojo/certs/client.crt';
-  local $ENV{MOJO_KEY_FILE}  = 't/mojo/certs/client.key';
-  local $ENV{MOJO_INSECURE}  = 0;
+my $ua;
```

Comment:
Variable outside of subtest.

Reply:
variable shared between tests and moved out of subtest.  How best to isolate?